### PR TITLE
composer: allow Symfony 3

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -29,11 +29,11 @@
         "php": ">=5.4",
         "doctrine/common": "~2.4",
         "doctrine/orm": "~2.4",
-        "symfony/http-kernel": "~2.4",
-        "symfony/dependency-injection": "~2.4"
+        "symfony/http-kernel": "~2.4|~3.0",
+        "symfony/dependency-injection": "~2.4|~3.0"
     },
     "require-dev": {
-        "symfony/framework-bundle": "~2.4, >=2.4.8",
```
Comment (mmoreram) [inline composer.json]: In fact this should be... `^2.4.8|^3.0`

File: composer.json [hunk 2]
```diff
@@ -29,11 +29,11 @@
         "php": ">=5.4",
         "doctrine/common": "~2.4",
         "doctrine/orm": "~2.4",
-        "symfony/http-kernel": "~2.4",
```
Comment (mmoreram) [inline composer.json]: Can you change this to `^2.4|^3.0`?
Here is the same, but I feel more comfortable working with `^` :)

File: composer.json (hunk 1 above)
Comment (TomasVotruba) [inline composer.json]: That would not allow installing Symfony ~2.5.

File: composer.json (hunk 2 above)
Comment (TomasVotruba) [inline composer.json]: Sure :)

File: composer.json (hunk 1 above)
Comment (mmoreram) [inline composer.json]: Why not?
`^2.4.8` === `>=2.4.8, <3.0.0`

Comment (TomasVotruba) [inline composer.json]: I think ~ is same ^ just preferring different higher to lower. But not sure, let me check composer docs.

Comment (mmoreram) [inline composer.json]: Nopes.
~ takes in account the level of the version.
~2.4 => until 3.0
~2.4.3 => until 2.5

^ doesn't care the level of the version
^2.4 => until 3.0
^2.4.3 => until 3.0

**Comment by mmoreram** [comment]
@TomasVotruba Thanks :D

**Comment by TomasVotruba** [comment]
Thanks

**Pull request merged**